src/components/AudioPlayer/AudioPlayer.module.scss

```diff
@@ -24,12 +24,13 @@
   @include breakpoints.tablet {
     height: constants.$audio-player-default-desktop-height;
   }
+  will-change: transform;
```

Comment:
There are a bunch of warning about will change: https://developer.mozilla.org/en-US/docs/Web/CSS/will-change

I think that using it is still the right call given how frequently this animation happens.